### ISSUE TITLE
Use sklearn train_test_split in place of StratifiedShuffleSplit

### ISSUE THREAD

**Issue by rhiever** [rhiever/tpot#99]
The latest version of sklearn's [train_test_split](http://scikit-learn.org/stable/modules/generated/sklearn.cross_validation.train_test_split.html) automatically stratifies the data by class(es) and shuffles the data prior to the split. Thus, we don't need to use the long-form version with [StratifiedShuffleSplit](http://scikit-learn.org/stable/modules/generated/sklearn.cross_validation.StratifiedShuffleSplit.html).

Let's replace all of the StratifiedShuffleSplit calls with train_test_split.

**Comment by pronojitsaha** [comment]
Ok..will look into this.

**Comment by rhiever** [comment]
Ping. How's this coming along? Should I mark it as "being worked on"?

**Comment by pronojitsaha** [comment]
Yes, I did give it a try couple of days back, but getting some spurious results. Checking on it. yes you can mark it.

**Comment by pronojitsaha** [comment]
Have sent across the PR. Thanks.
